django/thunderstore/core/utils.py

```diff
@@ -51,13 +51,6 @@ def ensure_fields_editable_on_creation(readonly_fields, obj, editable_fields):
         return list(x for x in readonly_fields if x not in editable_fields)
 
 
-class CommunitySiteSerializerContext:
```

Comment:
I suggest instead of removing this, we should add the community identifier to the context in similar fashion (see reasoning in other comments)

Reply:
CommunityMixin get_serializer_context to the CommunityMixin 👍